jni/src/test/java/com/horizen/schnorrnative/SchnorrKeyPairTest.java

```diff
@@ -17,6 +17,22 @@ public void testGenerate() throws Exception {
         }
     }
 
+    @Test
+    public void testDeriveFromSeed() throws Exception {
+        byte[] seed = { 1, 2, 3, 4, 5, 6, 7, 8 };
```

Comment:
Can we hardcode also (pk, sk) and check the code generates the same ?

Reply:
Sure will add.